lib/etc/openturns.conf.in

```diff
@@ -81,9 +81,9 @@
   <Drawable-DefaultPointStyle   value_str="plus"           />
   <Drawable-DefaultSurfaceColor value_str="white"          />
   <Drawable-NoSpecifiedLabel    value_str=""               />
-  <Drawable-DataThreshold       value_int="2000" />
-  <Drawable-DefaultPalettePhase value_int="12"   />
-
+  <Drawable-DataThreshold       value_int="2000"           />
+  <Drawable-DefaultPalettePhase value_int="12"             />
+  <Drawable-DefaultPaletteName  value_str="Tableau"        />
```

Comment:
Please insert the new keys according to the rules: bool first, then float, then str, then int, and sort the keys in alphabetical order within a given type

Reply:
Ok. Done.